composer.json

```diff
@@ -29,7 +29,8 @@
     "require-dev": {
         "psr/log": "~1.0",
         "phpcr/phpcr-api-tests": "2.1.4",
-        "symfony/console": "~2.0"
+        "phpcr/phpcr-benchmarks": "~1.0",
+        "phpunit/phpunit": "4.7.*"
```

Comment:
The benchmark suite depends on the PHPUnit bootstraping, which apparently needs some phpunit classes..